infra/aws/terraform/sig-release/patch-release-notification/versions.tf

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+terraform {
+  backend "s3" {
+    bucket = "tf-state-sig-release"
+    key    = "cherry-pick-notification"
+    region = "us-west-2"
+  }
+
+  required_version = "1.8.0"
```

Comment:
Can we relax the version if we don't have any dependency on 1.8.0?

Reply:
sure